src/client/components/Editor/EditorInput.js

```diff
@@ -11,7 +11,7 @@ import './EditorInput.less';
 
 class EditorInput extends React.Component {
   static propTypes = {
-    initialValue: PropTypes.string,
+    value: PropTypes.string, // eslint-disable-line react/require-default-props
```

Comment:
We have to disable this rule. If we set `defaultProp` on it, antd will complain about updating value without `setFieldsValue`.